aws/ami/files/scylla_install_ami

```diff
@@ -167,3 +167,22 @@ if __name__ == '__main__':
                 f.write(u'exclude=kernel kernel-devel')
         elif platform.machine() == 'aarch64':
             run('yum -y --enablerepo=centos-kernel update kernel')
+
+    if distro == 'centos':
+        run('yum install -y yum-utils')
+        packages = subprocess.check_output('repoquery --requires --resolve --recursive scylla', stderr=subprocess.STDOUT, shell=True)
+        with open('{}/scylla-packages.txt'.format(homedir), 'w') as f:
+            f.write(packages)
+    else:
+        deps = subprocess.check_output('apt-cache depends --recurse --no-recommends --no-suggests --no-conflicts --no-breaks --no-replaces --no-enhances --installed scylla', stderr=subprocess.STDOUT, shell=True, encoding='utf-8').splitlines()
+        pkgs=[]
+        for d in deps:
+            if re.match(r'^\w', d) and not re.match(r'.+:i386$', d) and d not in pkgs:
```

Comment:
just wondering, why do you filter out i386 packages? (and not filter el7.i686 for CentOS?)

Reply:
On Ubuntu, there is no tool to easily generate recursive dependency list, but we almost succeeded to do that using `apt-cache depends`, and filter out the output.
However, the output contains not installed i386 version of packages even I specified option to `--installed`.
I verified all listed packages are installed, and seems like only i386 versions are not installed: 
```
$ for p in `apt-cache depends --recurse --no-recommends --no-suggests --no-conflicts --no-breaks --no-replaces --no-enhances --installed scylla|grep "^\w"`;do dpkg -l $p > /dev/null;done
dpkg-query: no packages found matching procps:i386
dpkg-query: no packages found matching util-linux:i386
dpkg-query: no packages found matching login:i386
dpkg-query: no packages found matching openssl:i386
dpkg-query: no packages found matching dpkg:i386
dpkg-query: no packages found matching coreutils:i386
dpkg-query: no packages found matching libpam-modules-bin:i386
dpkg-query: no packages found matching tar:i386
dpkg-query: no packages found matching install-info:i386
```

Probably amd64 version of them are in dependency list, and installed.
But we don't need i386 version.
I guess there is an apt parameter to stop showing up them, but I haven't find it yet.
So the script is workaround of this problem.

Reply:
Ah, I found another problem on CentOS version.
CentOS version does not has problem we have on Ubuntu version, but we forgetting to specify --installed option to repoquery.
So CentOS version output is including some of not installed packages.

Reply:
Pushed fix for CentOS version. (add --installed)